Sources/Kanna/libxmlHTMLNode.swift

```diff
@@ -142,6 +142,12 @@ final class libxmlHTMLNode: XMLElement {
         self.weakDocument = document
         self.docPtr       = docPtr
         self.nodePtr      = xmlDocGetRootElement(docPtr)
+        if let x = xmlDocGetRootElement(docPtr) {
+            self.nodePtr = x
+        } else {
+            print("Is nil!")
```

Comment:
I don't allow the use of print statements in this library. We should handle this error as an exception.
I thought this had to be done by myself as it had to be carefully considered so as not to affect the user.

Reply:
Oh, you said "It still crashes with the following code. --> try? Kanna.HTML(html: " ", encoding: String.Encoding.utf8)" So I don't understand. Does my commit fix it?

Reply:
I said the problem didn't fix because you forgot to erase line 144. Without this mistake, your fix is ​​correct.
And I said it was a bit complicated because I had to devise error handling.

Reply:
Oh damn. I indeed did forget to erase line 144. I was going to remove it but must have just shut Xcode then. If only I erased line my first pull request ever would have been 100% correct.  🤣 Also great that you devised error handling. Error handling is not at all complicated for me as I have experience writIng it in my code but you have more experience so your code would perhaps be better.  Looking forward to helping with improving Kanna in the future!